
fix linking, don't link the Qt world via QT_LIBRARIES (including QtWebkit), only use QT_DECLARATIVE_LIBRARIES
consistently use QT_IMPORTS_DIR

diff -up libqzeitgeist-0.8.0/declarative/CMakeLists.txt.declarative libqzeitgeist-0.8.0/declarative/CMakeLists.txt
--- libqzeitgeist-0.8.0/declarative/CMakeLists.txt.declarative	2011-10-27 09:08:58.000000000 -0500
+++ libqzeitgeist-0.8.0/declarative/CMakeLists.txt	2011-10-31 07:36:01.751921806 -0500
@@ -14,7 +14,7 @@ include_directories(${CMAKE_CURRENT_SOUR
 include_directories(${CMAKE_CURRENT_SOURCE_DIR}/../src/)
 automoc4_add_library(QZeitgeistDeclarativePlugin MODULE ${declarative_SRCS})
 
-target_link_libraries(QZeitgeistDeclarativePlugin ${QT_LIBRARIES} qzeitgeist)
+target_link_libraries(QZeitgeistDeclarativePlugin ${QT_QTDECLARATIVE_LIBRARIES} qzeitgeist)
 
-install(TARGETS QZeitgeistDeclarativePlugin DESTINATION lib${LIB_SUFFIX}/qt4/imports/org/gnome/zeitgeist)
+install(TARGETS QZeitgeistDeclarativePlugin DESTINATION ${QT_IMPORTS_DIR}/org/gnome/zeitgeist)
 install(FILES qmldir DESTINATION ${QT_IMPORTS_DIR}/org/gnome/zeitgeist)
